src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -112,10 +114,17 @@ public async Task<IActionResult> GetHostStatus([FromServices] IScriptHostManager
 
         [HttpGet]
         [HttpPost]
+        [HttpOptions] // TEMP - allow Options only for testing
```

Comment:
FE changes have been made so scale pings will go to this endpoint. This OPTIONS support here is for the current prod testing I'm doing, since the FE currently sends an OPTIONS request to root. In my testing, I'm using a temporary web.config rewrite rule to rewrite the root pings to admin/host/ping. I won't check this [HttpOptions] change in.